app/src/main/java/swati4star/createpdf/fragment/PPTtoPdfFragment.java

```diff
@@ -0,0 +1,207 @@
+        mMorphButtonUtility = new MorphButtonUtility(mActivity);
+        ButterKnife.bind(this, rootview);
+        mMorphButtonUtility.morphToGrey(mCreatePdf, mMorphButtonUtility.integer());
+        mCreatePdf.setEnabled(false);
+        return rootview;
+    }
+
+    @Override
+    public void onAttach(Context context) {
+        super.onAttach(context);
+        mActivity = (Activity) context;
+        mFileUtils = new FileUtils(mActivity);
+    }
+
+    @OnClick(R.id.select_pptFile)
+    public void selectPPTFile() {
+        if (!mButtonClicked) {
+            Uri uri = Uri.parse(Environment.getRootDirectory() + "/");
+            Intent intent = new Intent(Intent.ACTION_GET_CONTENT);
+            intent.setDataAndType(uri, "*/*");
+            intent.addCategory(Intent.CATEGORY_OPENABLE);
+            try {
+                startActivityForResult(
+                        Intent.createChooser(intent, String.valueOf(R.string.select_file)),
+                        mFileSelectCode);
+            } catch (android.content.ActivityNotFoundException ex) {
+                showSnackbar(mActivity, R.string.install_file_manager);
+            }
+            mButtonClicked = true;
+        }
+    }
+
+    /**
+     * This function opens a dialog to enter the file name of
+     * the converted file.
+     */
+    @OnClick(R.id.create_PPTPdf)
+    public void openPPTtoPdf() {
+        if (!mPermissionGranted) {
+            getRuntimePermissions();
+            return;
+        }
+        new MaterialDialog.Builder(mActivity)
+                .title(R.string.creating_pdf)
+                .content(R.string.enter_file_name)
+                .input(getString(R.string.example), null, (dialog, input) -> {
+                    if (StringUtils.isEmpty(input)) {
+                        showSnackbar(mActivity, R.string.snackbar_name_not_blank);
+                    } else {
+                        final String inputName = input.toString();
+                        if (!mFileUtils.isFileExist(inputName + getString(R.string.pdf_ext))) {
+                            convertToPdf(inputName);
+                        } else {
+                            MaterialDialog.Builder builder = createOverwriteDialog(mActivity);
+                            builder.onPositive((dialog12, which) -> convertToPdf(inputName))
+                                    .onNegative((dialog1, which) -> openPPTtoPdf())
+                                    .show();
+                        }
+                    }
+                }).show();
+
+    }
+
+    @Override
+    public void onActivityResult(int requestCode, int resultCode, Intent data) {
+        mButtonClicked = false;
+        switch (requestCode) {
+            case mFileSelectCode:
+                if (resultCode == RESULT_OK) {
+                    mPPTFileUri = data.getData();
+                    mRealPath = RealPathUtil.getRealPath(getContext(), data.getData());
```

Comment:
@Swati4star @vyankatesh24 Here is the assigning/definition of `mRealPath`

Reply:
Have a look over return value.

Reply:
I logged the return value. It was `null`